src/main/java/seedu/taskell/commons/core/Config.java

```diff
@@ -61,7 +61,6 @@ public void setTaskManagerName(String taskManagerName) {
         this.taskManagerName = taskManagerName;
     }
 
-
     @Override
     public boolean equals(Object other) {
         if (other == this){
```

Comment:
Space is needed after brackets <br>

![image](https://cloud.githubusercontent.com/assets/13817632/19619853/8b344c30-98a2-11e6-91bf-74037b696688.png)